Automattic/wp-calypso
# [WIP]  Migrate lib/countries-list to Redux

Fixes #7920

Converts the `lib/countries-list` to a Redux State and integrates the new implementation for the Payments and Domains country lists.

Tests: (incomplete)
`npm run test-client client/state/countries`

User Testing:
- Check `blocks/credit-card-form` at the URL `/payment-methods/add-credit-card`
- Check `my-sites/upgrades/checkout/domain-details-form` by trying to add a domain.
- Check`my-sites/upgrades/checkout/secure-payment-form` by starting a checkout with a credit card.
